pages/common/xmllint.md

```diff
@@ -0,0 +1,32 @@
+# xmllint
+
+> XML parser.
+> Also useful for detecting errors in XML code.
+
+- Return all nodes named {{foo}}:
+
+`xmllint --xpath "//{{foo}}" {{source_file.xml}}`
+
+- Return as string the contents of first node named {{foo}}:
+
+`xmllint --xpath "string//{{foo}}" {{source_file.xml}}`
+
+- Use other other xpath expressions for more options in navigating xml tree. NOTE: xpath is a syntax for navigating XML trees too extensive to be covered here:
```

Comment:
Yup absolutely. I've put in these changes! let me know if there is anything else I can do!